package.json

```diff
@@ -48,7 +48,9 @@
     "graphql": "^16.0.0",
     "immer": "^9.0.0",
     "nth-check": ">=2.0.1",
-    "set-value": ">=4.0.1"
+    "set-value": ">=4.0.1",
+    "hds-core": "3.8.0",
+    "hds-react": "3.8.0"
```

Comment:
Are these in `resolutions` accidentally?

Reply:
Nope.

The resolutions field in a package.json file allows you to specify an exact version of a package that should be used in your project, even if the package is not directly required by your project. This can be useful if you want to ensure that a specific version of a package is used throughout your project and all of its dependencies, even if some of those dependencies do not specify a version constraint for the package.

Reply:
That's not the problem. Let me ask it in another way: why are you using `resolutions` instead of upgrading the package? Because you haven't upgraded `react-helsinki-headless-cms` yet to HDS v3.8?

Reply:
-> https://github.com/City-of-Helsinki/kukkuu-ui/pull/570